src/deprecated.jl

```diff
@@ -0,0 +1,18 @@
+
+# Deprecate RodriguesVec => RotationVec
+struct RodriguesVec{T} end
+@deprecate RodriguesVec(x, y, z) RotationVec(x, y, z)
+@deprecate RodriguesVec(t::NTuple{9}) RotationVec(t::NTuple{9})
+@deprecate RodriguesVec(r::Rotation) RotationVec(r::Rotation)
+function RodriguesVec{T}(x, y, z) where T
+    Base.depwarn("`RodriguesVec` is deprecated, use `RotationVec` instead", :Type)
+    RotationVec{T}(x,y,z)
+end
+function Base.one(::Type{RodriguesVec})
+    Base.depwarn("`one(::Type{RodriguesVec})` is deprecated, use `one(::Type{RotationVec})` instead.", :one)
+    one(RotationVec)
+end
+function Base.one(::Type{RodriguesVec{T}}) where T
+    Base.depwarn("`one(::Type{RodriguesVec{T}})` is deprecated, use `one(::Type{RotationVec{T}})` instead.", :one)
+    one(RotationVec{T})
+end
```

Comment:
I think you may be able to replace all this tricky deprecation logic with a use of deprecate_binding?

(In any case what you've got here won't quite work, because people could have `RodriguesVec{T}` field inside a larger struct, and solving that without deprecate_binding is basically impossible.)

Does the following do the trick?

```julia
Base.@deprecate_binding RodriguesVec RotationVec
```